docs/section5/Rule5-15.md

```diff
@@ -52,6 +52,6 @@
 
           Case 2: Else if slab-on-grade floor F-factor matches Table G3.4: ```else if surface_construction_b.f_factor == target_f_factor: PASS```  
 
-          Case 3: Else: ```else: FAIL```  
+          Case 3: Else: ```else: RAISE_WARNING: "SLABS BELOW 24" BELOW GRADE IS NOT REGULATED."```  
 
```

Comment:
I would change the warning message as follows: "Baseline slab F-factor is not as expected for slabs that are less than 24" below grade. Verify that the slab is more than 24" below grade and is unregulated." Or maybe do FAIL and include this message saying that it can actually be OK?

Reply:
Updated.

Reply:
Following up our discussion during the bi-weekly calls, I suggest changing the outcome to FAIL and keepting the accompanying message.